datasource/src/test/scala/quasar/physical/s3/S3DatasourceModuleSpec.scala

```diff
@@ -22,16 +22,20 @@ import quasar.api.datasource.DatasourceError.AccessDenied
 import quasar.connector.ResourceError
 import quasar.contrib.scalaz.MonadError_
 
-import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.ExecutionContext
 
 import argonaut.Json
-import cats.effect.IO
+import cats.effect.{ContextShift, IO, Timer}
 import org.specs2.mutable.Specification
 import shims._
 
 class S3DatasourceModuleSpec extends Specification {
   import S3DatasourceModuleSpec._
 
+  implicit val cs: ContextShift[IO] = IO.contextShift(ExecutionContext.global)
```

Comment:
Was about to complain about this when I realized that it's a test class. Carry on!